web/webpack.config.js

```diff
@@ -44,14 +36,9 @@ const config = {
     new webpack.EnvironmentPlugin({
       API_URL: null
     }),
-    new ExtractTextPlugin('app.css')
+    new ExtractTextPlugin('app.css'),
+    new UglifyJSPlugin()
   ]
 };
 
-if (isProd) {
-  config.plugins.push(new UglifyJSPlugin());
-} else {
-  config.devtool = 'cheap-module-eval-source-map';
-}
-
```

Comment:
Since this config is only for production now, we can get rid of all these conditionals.

Reply:
ha, welp, consider my question above answered.